README.md

```diff
@@ -1,2 +1,15 @@
-# hw01
-Repository for first CFSS homework assignment
+#**Lucas Buchheim-Jurisson**  
+***
+###*Education*
+* Year: 4th (undergrad)
+* Major: Political Science
+* Minor: Astronomy & Astrophysics
+
+###*Basics*
+I was born in Chicago and went to the Lab Schools here in Hyde Park before coming to UChicago. On campus, I am a member of the Trott Business Program (formerly UCIB), Chicago Project on Security and Terrorism, and ~~play~~ played on the varsity soccer team. 
+
+###*Interests*
```

Comment:
Also add a line break between headers and the main body text. Like this:

```
### *Interests*

Though I spent most of my college career working in finance, recently I've become very interested in data science. Post-graduation, I hope to work in hedge funds or tech for a year or two before applying to grad school. 
```

This will help Markdown render the document as you intended. For things like creating lists, there must be a linebreak between the text above it and the first item in the list, otherwise the formatting will be wrong